src/lib/dnssd/TxtFields.h

```diff
@@ -47,6 +47,10 @@ static constexpr size_t kKeyRotatingDeviceIdMaxLength   = 100;
 static constexpr size_t kKeyPairingInstructionMaxLength = 128;
 static constexpr size_t kKeyPairingHintMaxLength        = 10;
 
+static_assert(kMaxDeviceNameLen == kKeyDeviceNameMaxLength,
```

Comment:
Where is kMaxDeviceNameLen coming from here?  This header does not include any headers that define it...

This static_assert should probably be in the place where we assume the two things to be equal.  Likely in Dnssd.cpp, which presumably includes both these headers (and if it doesn't, it should).

Reply:
Moved assert to Dnssd.cpp